PowerPointLabs/PowerPointLabs/ShortcutsLab/ConvertToPicture.cs

```diff
@@ -61,41 +62,33 @@ private static void ConvertToPictureForShape(PowerPoint.Shape shape, int origina
             {
                 Logger.LogException(e, "Chart cannot be rotated.");
             }
-            shape.Copy();
-            float x = shape.Left;
-            float y = shape.Top;
-            float width = shape.Width;
-            float height = shape.Height;
-            shape.Delete();
-            PowerPoint.Shape pic = PowerPointCurrentPresentationInfo.CurrentSlide.Shapes.PasteSpecial(PowerPoint.PpPasteDataType.ppPastePNG)[1];
-            pic.Left = x + (width - pic.Width) / 2;
-            pic.Top = y + (height - pic.Height) / 2;
-            pic.Rotation = rotation;
-            // move picture to original z-order
-            while (pic.ZOrderPosition > originalZOrder)
-            {
-                pic.ZOrder(Office.MsoZOrderCmd.msoSendBackward);
-            }
-            while (pic.ZOrderPosition < originalZOrder)
-            {
-                pic.ZOrder(Office.MsoZOrderCmd.msoBringForward);
-            }
-            pic.Select();
-        }
+            
+            // Have no choice to use deprecated method because ConvertToPicture does not use ActionFramework
```

Comment:
The sentence structure of the comment is informal.

May I suggest:
> Utilises deprecated PowerPointPresentation class as ConvertToPicture does not utilise ActionFramework

Reply:
I agree, thanks for the suggestion!